src/Components/Projects/Projects.tsx

```diff
@@ -17,6 +17,7 @@ const handleClose = () => setModalState("close");
   const showAllProjects =()=>{
     axios.get(`${baseUrl}/Project`,{headers:reqHeaders}).then((response)=>{
       setAllProjs(response?.data?.data)
+
     }).catch((error)=>{
       console.log(error);
```

Comment:
👍 Good work on the API call! 🚀 Consider improving error handling for a more robust solution. Keep it up!